src/actions/list_objects_v2.rs

```diff
@@ -47,8 +48,8 @@ pub struct ListObjectsV2Response {
     pub max_keys: Option<u16>,
     #[serde(rename = "CommonPrefixes", default)]
     pub common_prefixes: Vec<CommonPrefixes>,
-    // #[serde(rename = "EncodingType")]
-    // encoding_type: String,
+    #[serde(rename = "EncodingType")]
+    encoding_type: String,
```

Comment:
I'm wandering whether this should be an enum or not

Reply:
Like an `Option<String>`?

Since there is an official S3 specification, as far as I know, I have no idea.
What I can say is that DO, minio, and was all return the `encoding_type` set to `url`. But maybe some other providers don't.

I would say we keep it like that until someone gives us an implementation that doesn't work like that personally 😅